release build failed because it tried to publish for jdk9

```
:servo-apache:bintrayUpload FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':servo-apache:bintrayUpload'.
> Could not upload to 'https://api.bintray.com/content/netflixoss/maven/servo/0.12.18/com/netflix/servo/servo-apache/0.12.18/servo-apache-0.12.18.jar': HTTP/1.1 409 Conflict [message:Unable to upload files: An artifact with the path 'com/netflix/servo/servo-apache/0.12.18/servo-apache-0.12.18.jar' already exists]
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,9 +2,9 @@ language: java
 matrix:
   include:
   - jdk: oraclejdk8
-    env: BINTRAY_PUBLISH=true
+    env: GRADLE_PUBLISH=true
   - jdk: oraclejdk9
-    env: BINTRAY_PUBLISH=false
+    env: GRADLE_PUBLISH=false
 sudo: false
 install: ./installViaTravis.sh
 script: ./buildViaTravis.sh

